components/feature/toolbar/src/test/java/mozilla/components/feature/toolbar/WebExtensionToolbarFeatureTest.kt

```diff
@@ -94,22 +94,20 @@ class WebExtensionToolbarFeatureTest {
 
         val browserAction = BrowserAction(
             title = "title",
-            icon = mock(),
+            loadIcon = { mock() },
```

Comment:
nit: Does worth it adding extra test coverage for the icon loading section and the `iconJob?.cancel()`  ?